src/features/timeline/TimelineIndexPage.js

```diff
@@ -44,22 +42,30 @@ const CycleIndexPage = ({ entryId }) => {
     })
   }, [selectedDate])
 
+  const notHasPlacedPeriod =
+    !entries.length || !entries.filter((s) => s.tags.length).length
+
+  useEffect(() => {
+    if (notHasPlacedPeriod) {
+      navigate("/incomplete")
+    }
+  }, [notHasPlacedPeriod])
+
   return (
     <AppLayout>
-      <AppMainToolbar>
-        <DatePicker date={selectedDate} />
-        <IconButton
-          aria-label="Scroll to today"
-          onClick={(event) => {
-            navigate(`/timeline`)
-          }}
-          style={{ marginLeft: "auto" }}
-        >
-          <Today />
-        </IconButton>
-      </AppMainToolbar>
-
       <AppPage>
+        <AppMainToolbar>
+          <DatePicker date={selectedDate} />
+          <IconButton
+            aria-label="Scroll to today"
+            onClick={(event) => {
+              navigate(`/timeline`)
```

Comment:
**nitpick:** Use `TIMELINE.to` from `navigation`

The analytics gets messed up if there is not a trailing slash, so trying to always use the one from navigation to make sure its always correct.

Reply:
I guess this one is mine from before I started doing that ;)

Reply:
Updated to: onClick={() => navigate(TIMELINE.to)}